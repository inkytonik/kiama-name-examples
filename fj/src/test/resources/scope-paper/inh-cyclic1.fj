// inheritance/cyclic.spt:immediate inheritance cycle
class A extends A {
  A() { super(); }
}
