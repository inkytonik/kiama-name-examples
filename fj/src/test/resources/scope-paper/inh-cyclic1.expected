src/test/resources/scope-paper/inh-cyclic1.fj:2:1:error: superclass cycle from A
class A extends A {
^
